go.mod

```diff
@@ -21,25 +21,25 @@ require (
 	github.com/kube-object-storage/lib-bucket-provisioner v0.0.0-20221122204822-d1a8c34382f1
 	github.com/libopenstorage/secrets v0.0.0-20230417161723-1022cc4d5aeb
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.66.0
-	github.com/prometheus-operator/prometheus-operator/pkg/client v0.66.0
-	github.com/rook/rook/pkg/apis v0.0.0-20230622190335-1d3f99bc1455
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.67.1
+	github.com/prometheus-operator/prometheus-operator/pkg/client v0.67.1
+	github.com/rook/rook/pkg/apis v0.0.0-20230804145609-7aaf46e11243
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
 	github.com/sykesm/zap-logfmt v0.0.4
-	go.uber.org/zap v1.24.0
+	go.uber.org/zap v1.25.0
 	golang.org/x/sync v0.3.0
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.27.3
-	k8s.io/apiextensions-apiserver v0.27.3
-	k8s.io/apimachinery v0.27.3
-	k8s.io/cli-runtime v0.27.3
-	k8s.io/client-go v0.27.3
-	k8s.io/cloud-provider v0.27.3
-	k8s.io/utils v0.0.0-20230505201702-9f6742963106
-	sigs.k8s.io/controller-runtime v0.15.0
+	k8s.io/api v0.28.0-rc.0
+	k8s.io/apiextensions-apiserver v0.28.0-rc.0
+	k8s.io/apimachinery v0.28.0-rc.0
+	k8s.io/cli-runtime v0.28.0-rc.0
+	k8s.io/client-go v0.28.0-rc.0
+	k8s.io/cloud-provider v0.28.0-rc.0
```

Comment:
instead of going for RC, if this change can wait for 1 week we can have 1.28 here as its planned for next week https://www.kubernetes.dev/resources/release/

Reply:
And I think it should be today (Tuesday), so updating for v0.28.0 makes sense. :)

Reply:
update to v0.28.0